pkg/dataobj/consumer/config.go

```diff
@@ -27,4 +29,10 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	cfg.BuilderConfig.RegisterFlagsWithPrefix(prefix, f)
 	cfg.UploaderConfig.RegisterFlagsWithPrefix(prefix, f)
+
+	if cfg.IdleFlushTimeout <= 0 {
+		cfg.IdleFlushTimeout = 60 * 60 * time.Second // default to 1 hour
+	}
+
+	f.DurationVar(&cfg.IdleFlushTimeout, prefix+"idle-flush-timeout", cfg.IdleFlushTimeout, "The maximum amount of time to wait in seconds before flushing an object that is no longer receiving new writes")
```

Comment:
You can set the default value as the 3rd argument in the flag definition, if you wish